@vates/task/index.test.js

```diff
@@ -145,6 +145,36 @@ describe('Task', function () {
     })
   })
 
+  describe('#failure()', function () {
```

Comment:
```suggestion
  describe('failure()', function () {
```
Is the `#` meaning something else than the function being private?
You also used it on lines 277 and 293 for `describe('#start'` and ` describe('#success()'`

Reply:
I used it to denote that it is an instance method and not a class method.

I don't know were this notation comes from but I adopted it before `#` was used for private fields.